Sources/FronteggSwift/FronteggAuth.swift

```diff
@@ -372,6 +372,10 @@ public class FronteggAuth: ObservableObject {
         self.logger.info("refreshing token")
         let accessToken = self.accessToken ?? ""
         
+        if(self.refreshingToken){
```

Comment:
The refreshingToken flag is reset after the refresh, whether successful or not. This is good practice, but you should ensure it happens in a defer block to guarantee it resets even if an exception or early return occurs so we won't get stuck with refreshingToken == true

Reply:
changed to defer (equivalent of finally) for cleanup refreshingToken indicator